### PR TITLE
Feat: Added hamburger menu for mobile navbar

### PR DESCRIPTION
**Pull request by heydoyouknowme0**

### CODE REVIEW

File: styles/buttonStyles.module.css
Comment (GetPsyched) [inline styles/buttonStyles.module.css:1]: no way not to use CSS modules?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: styles/buttonStyles.module.css
Comment (heydoyouknowme0) [reply styles/buttonStyles.module.css:1]: writing this in tailwind makes it completely unreadable and nightmare if we want to change anything
there's also styled jsx but its basically the same with inferior support.
Can make custom tailwind classes too but that will make them globally scoped.

Comment (GetPsyched) [reply styles/buttonStyles.module.css:1]: I'm currently refactoring this to use Tailwind and it's not that bad. The only issue I'm having is that `aria-expanded:rotate-45` isn't working lol

File: app/[locale]/header.tsx [hunk 1]
```diff
@@ -126,11 +127,41 @@ export default async function Header({ locale }: { locale: string }) {
               <Link href={`/${locale}/login`}>{text.login}</Link>
             </Button>
           </li>
-          <li className="lg:hidden">
-            <IoMenu
-              className="rounded-md bg-primary-700 p-1 text-shade-light hover:cursor-pointer"
-              size={40}
-            />
+          <li className="z-30  font-semibold lg:hidden">
```
Comment (GetPsyched) [inline app/[locale]/header.tsx:130]: Why does this keep happening?
```suggestion
          <li className="z-30 font-semibold lg:hidden">
```

File: app/[locale]/header.tsx [hunk 2]
```diff
@@ -126,11 +127,41 @@ export default async function Header({ locale }: { locale: string }) {
...
+                        asChild
+                        className="text-left text-shade-dark"
+                        variant="link"
+                      >
+                        <Link href={`/${locale}/${href}`}>{label}</Link>
+                      </Button>
+                    </li>
+                  ))}
+                </ul>
+                <hr className="opacity-50" data-dropdownignore={true} />
```
Comment (GetPsyched) [inline app/[locale]/header.tsx:156]: This should also work
```suggestion
                <hr className="opacity-50" data-dropdownignore />
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/[locale]/header.tsx (hunk 1 above)
Comment (heydoyouknowme0) [reply app/[locale]/header.tsx]: ill be more careful next time, prettier doesnt remove those

**Pull request merged**